docs/sources/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+description: The GitHub data source lets you visualize GitHub data in Grafana dashboards.
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+menuTitle: GitHub data source
+title: GitHub data source plugin for Grafana
+weight: 10
+---
+
+# GitHub data source plugin for Grafana
+
+The GitHub data source plugin for Grafana lets you to query the GitHub API in Grafana so you can visualize your GitHub repositories and projects.
+
+{{< docs/play title="GitHub data source plugin demo" url="https://play.grafana.org/d/cdgx261sa1ypsa/3-single-repo-with-override-examples" >}}
+
+## Query types
+
+The plugin supports the following query types:
+
+- Commits
+- Issues
+- Contributors
+- Tags
+- Releases
+- Pull requests
+- Labels
+- Repositories
+- Milestones
+- Packages
+- Vulnerabilities
+- Projects
+- Stargazers
+- Workflows
+- Workflow usage
+
+## Supported features
+
+With the plugin you can:
+
+- Visualize queries
+- Use template variables
+- Configure Annotations
+- Cache queries
+
+## Caching
+
+Caching on this plugin is always enabled.
+
+{{< admonition type="note" >}}
+To work around [GitHub's rate limiting](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28), the plugin caches requests aggressively.
+
+This can mean that it takes up to five minutes for a new pull request, commit, or issue to show up in a query.
+{{< /admonition >}}
+
+## Requirements
+
+To use the GitHub data source plugin, you will need:
+
+- A free [GitHub](https://github.com/) or a [GitHub Enterprise](https://github.com/enterprise) account.
+- Any of the following Grafana editions:
+  - Grafana OSS server.
+  - A [Grafana Cloud](https://grafana.com/pricing/) stack.
+  - An on-premise Grafana Enterprise server with an [activated license](https://grafana.com/docs/grafana/latest/enterprise/license/activate-license/).
+
+## Get started
```

Comment:
@jdbaldry completed it.

Please review. 

Also, not 100% sure about the linking of the page as after this PR goes live. It probably goes to this URL; https://grafana.com/docs/plugins/github-datasource/latest/

Reply:
fixed now!!